### PR TITLE
Add v1.0.0 verifier Sepolia deployment to dev docs

### PR DESCRIPTION
**Pull request by nategraf**

### CODE REVIEW

File: website/api_versioned_docs/version-1.0/blockchain-integration/contracts/verifier.md
Comment (flaub) [inline website/api_versioned_docs/version-1.0/blockchain-integration/contracts/verifier.md:61]: This should point to 1.0

File: website/api/blockchain-integration/contracts/verifier.md
Comment (pdg744) [reply website/api/blockchain-integration/contracts/verifier.md:49]: or perhaps we can point to `latest`?

Comment (pdg744) [reply website/api/blockchain-integration/contracts/verifier.md:49]: ...or maybe there's a deployed version that makes more sense to point to?

File: website/api/blockchain-integration/contracts/verifier.md
Comment (pdg744) [inline website/api/blockchain-integration/contracts/verifier.md:48]: This link to 0.10 could use a refresh -- not sure what the best way to manage maintenance is on these. 

I think when it comes to cutting new zkVM versions we have some systems in place so that links get updated during the release process, but I don't think we have those systems set up yet for risc0-ethereum versioning.

**Comment by nategraf** [comment]
Should we go ahead and only display addresses for one version of the contracts per version of the docs?

**Comment by pdg744** [comment]
> Should we go ahead and only display addresses for one version of the contracts per version of the docs? In the case of "next" this might mean only adding addresses as part of the release process, when the next version is fixed and the contracts are deployed.

I'm not sure I follow your question/suggestion, but I'm happy to defer to you for how best to manage this on `next` -- seems like a tricky issue that you have a better handle on than I do.

**Comment by nategraf** [comment]
> > Should we go ahead and only display addresses for one version of the contracts per version of the docs? In the case of "next" this might mean only adding addresses as part of the release process, when the next version is fixed and the contracts are deployed.
> 
> I'm not sure I follow your question/suggestion, but I'm happy to defer to you for how best to manage this on `next` -- seems like a tricky issue that you have a better handle on than I do.

Essentially, the versioned pages would each have a table with one entry per contract, all at the current version (e.g. 1.0 or 1.1). If they go to the API docs for the "next" version, there would be no table, but a note about how we deploy contracts when the version is released.

File: website/api/blockchain-integration/contracts/verifier.md
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md:51]: I'll start adding a version string constant to the contracts.

**Comment by nategraf** [comment]
Changing the base to https://github.com/risc0/risc0/pull/1957, which should be run first to fix the CI check job and run `prettier`

File: website/api/blockchain-integration/contracts/verifier.md
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md]: Check out the latest version of this PR and let me know what you think

**Comment by nategraf** [comment]
@pdg744 this is ready for you to take a look at. I've simplified the table such that each versioned page only has the table of addresses for its own version.

**Review by pdg744 — APPROVED** [review]
Nice! This looks quite clean -- I like the way you handled `next` vs. `1.0` vs. older versions.

**Comment by pdg744** [comment]
I've approved the changes re: the actual copy for the website... I'm not super well-informed on the tests that you've changed here (and not sure whether there's a separate codeowner whose approval is required for those tests)

**Comment by nategraf** [comment]
Indeed, it looks like @flaub needs to approve this. I believe because I touched `.github/workflows/website.yml`

File: website/api/blockchain-integration/contracts/verifier.md
Comment (flaub) [inline website/api/blockchain-integration/contracts/verifier.md:70]: This needs to be a relative path I think

Comment (flaub) [reply website/api/blockchain-integration/contracts/verifier.md:70]: Basically, once we run the docusaurus version command to promote the next version, this URL will break. But if we keep it relative, it will work across promotions.

**Review by flaub — APPROVED** [review]

File: website/api/blockchain-integration/contracts/verifier.md
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md:70]: The intention here is to link to the latest release, rather than the "next" release that the reader is currently viewing. So that's why it's absolute. Does that seem right to you? I can open up a follow-up PR to fix that. (I enabled auto-merge before I saw this)

Comment (flaub) [reply website/api/blockchain-integration/contracts/verifier.md:70]: I wonder if we should be putting this kind of content that is 'versionless' under `docs` rather than the versioned `api`?

**Pull request merged**
